src/EventStore.Client.Streams/EventStoreClientServiceCollectionExtensions.cs

```diff
@@ -27,8 +40,24 @@ public static class EventStoreClientServiceCollectionExtensions {
 				options.ConnectivitySettings.Address = address;
 				options.CreateHttpMessageHandler = createHttpMessageHandler;
 			});
+#endif
 
-
+#if NETCOREAPP3_1
+		/// <summary>
+		/// Adds an <see cref="EventStoreClient"/> to the <see cref="IServiceCollection"/>.
+		/// </summary>
+		/// <param name="services"></param>
+		/// <param name="address"></param>
+		/// <param name="createHttpMessageHandler"></param>
+		/// <returns></returns>
+		/// <exception cref="ArgumentNullException"></exception>
+		public static IServiceCollection AddEventStoreClient(this IServiceCollection services, Uri address,
+			// ReSharper disable once MethodOverloadWithOptionalParameter
+			Func<HttpMessageHandler>? createHttpMessageHandler = null)
+			=> services.AddEventStoreClient(options => {
+				options.ConnectivitySettings.Address = address;
```

Comment:
@thefringeninja same here, is options.CreateHttpMessageHandler = createHttpMessageHandler; missing?